deployment/bootstrap.sh

```diff
@@ -0,0 +1,79 @@
+# This scrips belongs to the Krustlet project and is a copy of what can be found
+# here: https://github.com/deislabs/krustlet/blob/main/docs/howto/assets/bootstrap.sh
```

Comment:
is this needed to be added if you reference krustlets instructions in the documentation? Why not just point to krustlets (local clone)?

Reply:
Thanks for the comment, that makes sense. Will be removing this file, it's not even mentioned on the demo.